packages/pn-personafisica-webapp/src/components/Contacts/AddSpecialContactDialog.tsx

```diff
@@ -214,107 +213,104 @@ const AddSpecialContactDialog: React.FC<Props> = ({
-                    ) : null,
-                }}
-              />
-            )}
-          </Stack>
-
-          <Stack my={2}>
-            <Typography variant="caption-semibold" mb={1}>
-              {t(`special-contacts.senders`, { ns: 'recapiti' })}
-            </Typography>
-            <ApiErrorWrapper
-              apiId={CONTACT_ACTIONS.GET_ALL_ACTIVATED_PARTIES}
-              mainText={t('special-contacts.fetch-party-error', { ns: 'recapiti' })}
-              reloadAction={getParties}
-            >
-              <PnAutocomplete
-                id="sender"
-                data-testid="sender"
-                size="small"
-                options={parties ?? []}
-                autoComplete
-                getOptionLabel={getOptionLabel}
-                noOptionsText={t('common.enti-not-found', { ns: 'recapiti' })}
-                isOptionEqualToValue={(option, value) => option.id === value.id}
-                onChange={senderChangeHandler}
-                inputValue={formik.values.sender.name}
-                disabled={!!value || sender.senderId !== 'default'}
-                onInputChange={(_event, newInputValue, reason) => {
-                  if (reason === 'input') {
-                    void formik.setFieldTouched('sender', true, false);
-                    void formik.setFieldValue('sender', { id: '', name: newInputValue });
-                  }
-                }}
-                filterOptions={(e) => e}
-                renderOption={renderOption}
-                renderInput={(params) => (
-                  <TextField
-                    {...params}
-                    name="sender"
-                    label={entitySearchLabel}
-                    error={formik.touched.sender && Boolean(formik.errors.sender?.name)}
-                    helperText={formik.touched.sender && formik.errors.sender?.name}
-                  />
-                )}
-                sx={{ flexGrow: 1, flexBasis: 0 }}
-              />
-            </ApiErrorWrapper>
-          </Stack>
+              fullWidth
+              id="s_value"
+              name="s_value"
+              label={t(`special-contacts.link-${formik.values.channelType.toLowerCase()}-label`, {
+                ns: 'recapiti',
+              })}
+              value={formik.values.s_value}
+              onChange={handleChangeTouched}
+              error={formik.touched.s_value && Boolean(formik.errors.s_value)}
+              helperText={formik.touched.s_value && formik.errors.s_value}
+              InputProps={{
+                startAdornment:
+                  formik.values.channelType === ChannelType.SMS ? (
+                    <InputAdornment position="start">{internationalPhonePrefix}</InputAdornment>
+                  ) : null,
+              }}
+              sx={{ mb: 2 }}
+            />
+          )}
+          <Typography variant="caption-semibold" mb={2}>
```

Comment:
It seems that mb is not being applied correctly. Additionally, it should be set to 1, not 2.